Sources/Model/PhotoCollectionViewDataSource.swift

```diff
@@ -63,18 +63,23 @@ final class PhotoCollectionViewDataSource : NSObject, UICollectionViewDataSource
         if let settings = settings {
             cell.settings = settings
         }
-        
+
+        // Clear any previously shown image
+        cell.imageView.image = nil
```

Comment:
This prevents the previous image being seen if a request is delayed, which can happen when scrolling quickly through a large album, that is fetching images from the cloud.

Reply:
I'll move this to PhotoCells prepareForReuse